README.md

```diff
@@ -59,10 +59,13 @@ KTF builds and runs on Fedora, but you will need to tweak some of the commands.
 following content (tested with Fedora 32):
 
 ```
+FEDORA=$(shell grep NAME=Fedora /etc/os-release)
```

Comment:
Maybe let's call the variable: DIST_NAME and use something like: awk -F= '/NAME=/ {print $NF}' ?

Reply:
ok